app/src/main/java/com/yagni/view/ManageInvGUI.java

```diff
@@ -9,365 +9,447 @@
 import com.yagni.controller.*;
```

Comment:
Remove line 8 whitespace, you can group all the imports together no need for that space.

Reply:
Removed line 8 whitespace!